.github/workflows/pull-request.yml

```diff
@@ -33,10 +33,21 @@ jobs:
       contents: read
     steps:
       - uses: actions/checkout@v3
+      - name: setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
       - uses: actions/setup-python@v4
         with:
           python-version: '3.11'
           cache: 'pip' # caching pip dependencies
       - run: pip install -r requirements.txt
       - name: Test Django
-        run: python ./manage.py test */tests
+        run: pytest whereami/tests
+      - name: generate spec.yaml
+        run: python ./manage.py spectacular --color --file ./angular/whereami/src/app/api/generated/spec.yaml --validate

```

Comment:
@sigmea94 spectacular generates a completely different schema in the github pipeline:
[diff.patch](https://github.com/martilidad/whereami_server/files/11188713/diff.patch)

Even after `pip install --force-reinstall --no-cache-dir -r requirements.txt` I do not get these results. 
I actually like the generated schema code & client from the pipeline much more, but can't reproduce it locally.  
Any ideas?

Reply:
The tags are simply created differently when running spectacular on linux.... :/ 
Tested with a docker image. 
One solution could be to tag manually.